### PR TITLE
LPS-189589:Fix PropertiesSQLStylingCheck for catch duplicated parenthesis

### PR DESCRIPTION
**Pull request by qz1010**

### CODE REVIEW

**Comment by qz1010** [comment]
ci:test:sf

**Comment by qz1010** [comment]
ci:test:relevant

File: modules/apps/frontend-taglib/frontend-taglib-clay/test.properties [hunk 1]
```diff
@@ -11,24 +11,26 @@
     test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
         (testray.component.names == "Clay") OR \
         (\
```
Comment (ling-alan-huang) [inline modules/apps/frontend-taglib/frontend-taglib-clay/test.properties:13]: Extra parenthesis?

Comment (ling-alan-huang) [inline modules/apps/frontend-taglib/frontend-taglib-clay/test.properties:13]: Extra parenthesis?

Comment (ling-alan-huang) [reply modules/apps/frontend-taglib/frontend-taglib-clay/test.properties:13]: Line 13, Line 35

**Comment by ling-alan-huang** [comment]
see https://github.com/ling-alan-huang/liferay-portal/pull/1976

File: modules/apps/frontend-taglib/frontend-taglib-clay/test.properties (hunk 1 above)
Comment (qz1010) [reply modules/apps/frontend-taglib/frontend-taglib-clay/test.properties:13]: The previous expression is connected with OR, followed by AND, and the last two expressions are connected with AND to form a group. However, due to the previous SF, the parentheses have been removed.
AND and OR appearing at the same level can affect readability and are prone to errors

**Comment by qz1010** [comment]
see #1981

**Pull request closed without merging**